### DISCUSSION TITLE
Floating bolt next to Eng 1's cowling

### DISCUSSION THREAD

**Discussion by Allanzkie** [flybywiresim/aircraft#9378] [Help]
Hello, apologies if this is the wrong category or if this has been posted before please feel free to remove it.

There seems to be a floating bolt next to the engine cowl of Engine No. 1

That's all thank you.

![image fbw2](https://github.com/user-attachments/assets/fbba994e-5aab-4adf-a644-50fd32d1a579)
![image fbw](https://github.com/user-attachments/assets/ce444414-869d-4bce-b122-7a0ecf3c89d6)

**Comment by Maximilian-Reuter** [comment]
Hi thankyou for your report, would it be possible for you to open this again but this time as an issue ? Discussions arent really designed to track issues.

**Reply by Allanzkie** [reply]
Hello, unfortunately it's my first time using Github so I'm not exactly sure how to do that. I only posted here because someone said in the Discord A380 Support channel told me to escalate it to Github.

EDIT: I found the tab lol all good now.
